lib/statsd/instrument/environment.rb

```diff
@@ -140,19 +140,25 @@ def default_sink_for_environment
         case environment
         when "production", "staging"
           connection = if statsd_uds_send?
-            StatsD::Instrument::UdsConnection.new(statsd_socket_path)
+            StatsD::Instrument::UdsConnection.new(
+              statsd_socket_path,
+              max_packet_size: statsd_max_packet_size.to_i,
+            )
           else
             host, port = statsd_addr.split(":")
-            StatsD::Instrument::UdpConnection.new(host, port.to_i)
+            StatsD::Instrument::UdpConnection.new(
+              host,
+              port.to_i,
+              max_packet_size: statsd_max_packet_size.to_i,
```

Comment:
I think this `to_i` is redundant, no? It looks to me like `statsd_max_packet_size` always returns an integer according to how you've defined it above.